beacon_node/network/src/beacon_processor/worker/gossip_methods.rs

```diff
@@ -734,16 +735,23 @@ impl<T: BeaconChainTypes> Worker<T> {
                 self.send_sync_message(SyncMessage::UnknownBlock(peer_id, block));
                 return None;
             }
+            Err(e @ BlockError::BeaconChainError(_)) => {
+                debug!(
+                    self.log,
+                    "Gossip block beacon chain error";
+                    "error" => ?e,
+                );
```

Comment:
For consistency I think we should we also propagate an `Ignore` here? What does the network stack do if we never get back to it about a message?

Reply:
I think nothing is the same as in `Ignore`, but I'll add it here for consistency. Thanks!